automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -209,6 +235,192 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLike w
 
     }
 
+    "should be able to create workspace and run sub-workflow tasks in non-US regions" in {
```

Comment:
TOL (the answer may well be "no"): is there enough similarity between these multiple tests to extract a common pattern?

Reply:
Some portion of the code could be extracted but it didn't seem that the result would be worth the effort since it is test code.